### PR TITLE
Github actions

### PR DESCRIPTION
**Pull request by mjoyce91**
Some fun github actions to add labels to PRs:

- chore/feature/fix label
- t-shirt size based on lines of code
- approval status

### CODE REVIEW

File: .github/workflows/size-label.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: size-label
+        uses: "pascalgn/size-label-action@v0.4.3"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        with:
+          sizes: >
+            {
+              "0": "XS",
```
Comment (elizabeth-jimenez) [inline .github/workflows/size-label.yml:14]: I like these sizes lol - i went through some of my old PRs and was like guessed at the size before looking at the lines

File: .github/workflows/pr-review.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    name: Label when approved
+    runs-on: ubuntu-latest
+    steps:
+    - name: Label when approved
+      uses: pullreminders/label-when-approved-action@master
+      env:
+        APPROVALS: "1"
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        ADD_LABEL: "approved"
+        REMOVE_LABEL: "awaiting%20review"
```
Comment (elizabeth-jimenez) [inline .github/workflows/pr-review.yml:14]: i just dont see the awaiting review label that we are asking to be removed

**Review by elizabeth-jimenez — CHANGES_REQUESTED** [review]
testing

**Review by elizabeth-jimenez — APPROVED** [review]

File: .github/workflows/pr-review.yml (hunk 2 above)
Comment (mjoyce91) [reply .github/workflows/pr-review.yml:14]: Perhaps we're supposed to add that ourselves?

**Pull request merged**